README.md

```diff
@@ -230,6 +231,7 @@ cog.out(pretty_list)
 | [<img src=".github/github.png" align="top" width="20" />](https://ludo.libretro.com/) | `ludo` | <i>A minimalist frontend for emulators.</i> |
 | [<img src=".github/launchpad.png" align="top" width="20" />](https://lutris.net/) | `lutris` | <i>Open Gaming Platform.</i> |
 | [<img src=".github/github.png" align="top" width="20" />](https://getmailspring.com/) | `mailspring` | <i>Boost your productivity and send better email with the best mail client.</i> |
+| [<img src=".github/direct.png" align="top" width="20" />](https://code-industry.net/free-pdf-editor/) | `master-pdf-editor` | <i>Edit PDF files in Linux. It enables you to create, edit, view and print interactive PDF documents.</i> |
```

Comment:
The name of the package used here does not match the name of the package used in the `deb_` function. Was this section generated by [cog](https://pypi.org/project/cogapp)?